recipe/meta.yaml

```
@@ -1,61 +1,204 @@
+    {% set tests_to_skip = tests_to_skip + " or test_repr_html_not_callable" %}  # [win]
+
+    # AttributeError: module 'numpy' has no attribute 'bool8'. Did you mean: 'bool'?
+    {% set ignore_tests = ignore_tests + " --ignore=streamlit_tests/lib/tests/streamlit/delta_generator_test.py" %}  # [py>312]
+    {% set ignore_tests = ignore_tests + " --ignore=streamlit_tests/lib/tests/streamlit/elements/plotly_chart_test.py" %}  # [py>312]
+    {% set ignore_tests = ignore_tests + " --ignore=streamlit_tests/lib/tests/streamlit/runtime/caching/cache_data_api_test.py" %}  # [py>312]
+    {% set ignore_tests = ignore_tests + " --ignore=streamlit_tests/lib/tests/streamlit/runtime/caching/cache_resource_api_test.py" %}  # [py>312]
+    {% set ignore_tests = ignore_tests + " --ignore=streamlit_tests/lib/tests/streamlit/runtime/fragment_test.py" %}  # [py>312]
+    {% set ignore_tests = ignore_tests + " --ignore=streamlit_tests/lib/tests/streamlit/runtime/state/widgets_test.py" %}  # [py>312]
+    {% set ignore_tests = ignore_tests + " --ignore=streamlit_tests/lib/tests/streamlit/streamlit_test.py" %}  # [py>312]
+
+    # AssertionError: 404 != 200
+    {% set ignore_tests = ignore_tests + " --ignore=streamlit_tests/lib/tests/streamlit/streamlit_test.py" %}
+    {% set ignore_tests = ignore_tests + " --ignore=streamlit_tests/lib/tests/streamlit/web/server/server_test.py" %}
+
+    # sqlite3.OperationalError: unable to open database file
+    {% set tests_to_skip = tests_to_skip + " or test_verify_sqlite3_integration" %}  # [win]
+
+    # PermissionError: [Errno 13] Permission denied: 'C:\\cygwin64\\tmp\\tmp9jhk3u1s.vtt'
+    {% set tests_to_skip = tests_to_skip + " or test_st_video_subtitles_path" %}  # [win]
+
+    # UnicodeEncodeError: 'charmap' codec can't encode character '\U0001f6a8' in position 83: character maps to <undefined>
+    {% set tests_to_skip = tests_to_skip + " or test_alert" %}  # [win]
+
+    # NotADirectoryError: [WinError 267] The directory name is invalid: 'C:\\cygwin64\\tmp\\tmpb98enk00\\tmp6o52qgcdstylesheet.css'
+    {% set tests_to_skip = tests_to_skip + " or test_mimetype_is_overridden_by_server" %}  # [win]
+    {% set tests_to_skip = tests_to_skip + " or test_nonexistent_urls_return_default_page" %}  # [win]
+    {% set tests_to_skip = tests_to_skip + " or test_parse_url_path_200" %}  # [win]
+    {% set tests_to_skip = tests_to_skip + " or test_reserved_paths_serve_404" %}  # [win]
+
+    # No such component directory: 'C:\a\test\component\directory'
+    {% set tests_to_skip = tests_to_skip + " or test_register_component_with_path" %}  # [win]
+    {% set tests_to_skip = tests_to_skip + " or test_register_duplicate_path" %}  # [win]
+
+    # IndexError: pop from empty list
+    {% set ignore_tests = ignore_tests + " --ignore=streamlit_tests/lib/tests/streamlit/runtime/runtime_test.py" %}  # [win]
+
+    test:
+      source_files:
+        - streamlit_tests/lib/tests
+        - streamlit_tests/lib/setup.py
```

```suggestion
        - streamlit_tests/lib/setup.py
        - streamlit_tests/lib/pytest.ini
```
Test configuration is here:
https://github.com/streamlit/streamlit/blob/develop/lib/pytest.ini#L1-L11